components/cam/src/physics/rrtmgp/radiation_utils.F90

```diff
@@ -333,7 +334,9 @@ subroutine handle_error(error_message, fatal)
          if (fatal_local) then
             call endrun(trim(error_message))
          else
-            print *, trim(error_message)
+            if (present(warn)) then
```

Comment:
Have you tested this? The way you have this setup now, it doesn't look like you ever actually test the value of `warn`, only if it's present. Thus, I expect that you will ALWAYS be throwing a warning, whether you set `warn` to `.true.` or `.false.` in the call. Instead, I suggest something like the following:
```
logical :: warn_local
warn_local = .true.
if (present(warn)) warn_local = warn
```
and then in the error checking block:
```
if (fatal_local) then
   call endrun(trim(error_message))
else if (warn_local) then
   print *, trim(error_message)
end if
```

Reply:
This appears to be a duplicate comment? I'll resolve for now.